entry.go

```diff
@@ -194,6 +194,7 @@ func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
 		claimInterval := time.Duration(te.config.ReverseClaimInterval) * time.Second
 		onErr := nkn.NewOnError(1, nil)
 		isClosed := false
+		reversePrice, _ := common.StringToFixed64(te.config.ReversePrice)
```

Comment:
1. should use `ParsePrice` instead of `common.StringToFixed64`
2. never ignore error